alert-policies/amazon-inspector/HighAssessmentRunFindings.yml

```diff
@@ -0,0 +1,27 @@
+name: High Assessment Run Findings
+
+description: |+
+  This alert is triggered if the number of Assessment Run Findings exceeds 100 for 10 minutes.
```

Comment:
```suggestion
  This alert is triggered if the number of Assessment Run Findings exceeds 100 for 5 minutes.
```

Reply:
@pkosanam , addressed, I have changed the duration.